File: gpao-tex-ui/source/main/src/app/services/product/models/produit-value.ts
/* tslint:disable */
/* eslint-disable */
import { CouleurValue } from '../models/couleur-value';
import { ElementTraitementValue } from '../models/element-traitement-value';
import { MesureProduitValue } from '../models/mesure-produit-value';
import { PilotageEfficienceValue } from '../models/pilotage-efficience-value';
import { ProduitGoshippementValue } from '../models/produit-goshippement-value';
import { TailleValue } from '../models/taille-value';
export interface ProduitValue {
  cadence?: number;
  codeConception?: string;
  couleurs?: Array<CouleurValue>;
  designation?: string;
  elementTraitements?: Array<ElementTraitementValue>;
  etat?: string;
  ficheBesoin?: boolean;
  id?: number;
  ligneProduitId?: number;
  mesureProduits?: Array<MesureProduitValue>;
  partieInteresseId?: number;
  pilotageEfficiences?: Array<PilotageEfficienceValue>;
  prixMajore?: number;
  prixUnitaire?: number;
  produitGoShippements?: Array<ProduitGoshippementValue>;
  quantite?: number;
  reference?: string;
  sansCouleur?: boolean;
  sansTaille?: boolean;
  siteId?: number;
  sousFamilleProduitId?: number;
  tailles?: Array<TailleValue>;
}

File: gpao-tex-core/ordre/src/main/java/com/gpro/consulting/ordre/dto/ProduitValue.java
package com.gpro.consulting.ordre.dto;

import java.math.BigDecimal;
import java.util.List;

public record ProduitValue(
        Long id,
        Long sousFamilleProduitId,
        Long siteId,
        Long themeId,
        Long ligneProduitId,
        List<ProduitGoshippementValue> produitGoShippements,
        List<MesureProduitValue> mesureProduits,
        List<TailleValue> tailles,
        List<CouleurValue> couleurs,
        List<PilotageEfficienceValue> pilotageEfficiences,
        List<ElementTraitementValue> elementTraitements,
        Long partieInteresseId,
        String reference,
        String designation,
        boolean ficheBesoin,
        boolean sansTaille,
        boolean sansCouleur,
        String codeConception,
        Integer quantite,
        Integer cadence,
        BigDecimal prixUnitaire,
        BigDecimal prixMajore,
        String etat
) {
}